### ISSUE TITLE
Setting default width for Description

### ISSUE THREAD

**Issue by KiqoCode** [alerta/alerta-webui#474]
**Is your feature request related to a problem? Please describe.**
In earlier versions, the sizing of columns seemed to be automatic for all user because description was always around 60% in width I would say. But since updated to 8.5.0, the width by default seems to be equal for all columns or close to. Can be changed on user level but it would be really nice to have a default setting for this since we have quite alot of users in Alerta, around 50 users. 

**Describe the solution you'd like**
A default setting for description width in alertad.conf

**Describe alternatives you've considered**
For each user to set it by themself, but that's not a nice solution for us.

**Comment by Dissmiss3d** [comment]
hello, after we updated, some places started to appear as links, how can we fix this?

![image](https://user-images.githubusercontent.com/85514498/121069073-babf6900-c7d5-11eb-9efb-b009a243e700.png)

**Comment by mouglou** [comment]
Agree, the new column sizing is not really user friendly, the different columns are too large and description field is not displaying the full message.

Any update ?
